src/foam/doc/DocBrowser.js

```diff
@@ -532,3 +559,954 @@ foam.debug.doc = function(opt_obj, showUnused) {
 
 // TODO:
 //    remove LinkView
+
+
+foam.CLASS({
+  package: 'foam.doc',
+  name: 'UMLDiagram',
+  extends: 'foam.u2.Element',
+
+  implements: ['foam.memento.MementoMgr'],
+  imports: ['browserPath'],
+  requires: [
+    'foam.doc.DocBorder',
+    'foam.graphics.Label',
+    'foam.graphics.Box',
+    'foam.u2.PopupView',
+    'foam.doc.Link',
+    'foam.doc.ClassLink',
+    'foam.graphics.Transform'
+  ],
+
+  exports: ['as data'],
+
+  constants: {
+    SELECTED_COLOR: 'white', //#ddd',
+    UNSELECTED_COLOR: '#FFFFCC'
+  },
+
+  axioms: [
```

Comment:
We've recently added a 'css' property to Model so you can more easily do this as just:
css: `
  css here
  and here
  ...
`,